WTWSMS/events/WTWSMS_historical_events.txt

```diff
@@ -997,3 +997,76 @@ narrative_event = {
 		prestige = 150
     }
 }
+
+# 526 Antioch earthquake
+narrative_event = {
+	id = hist.24
+	title = hist.24.name
+	desc = hist.24.desc
+	picture = GFX_evt_falling_statue
+	border = GFX_event_narrative_frame_diplomacy
+	
+	only_playable = yes
+	
+	major = yes
+
+	trigger = {
+		year = 526
+		NOT = { has_global_flag = 526_antioch_earthquake }
+		b_antiocheia = {
+			holder_scope = { top_liege = { character = ROOT } }
+		}
+	}
+	
+    mean_time_to_happen = {
+        months = 3
+	}
+	
+    option = {
+		name = hist.24.a
+		
+		trigger = {
+			b_antiocheia = {
+				holder_scope = { top_liege = { character = ROOT } }
+			}
+		}
+		
+		b_antiocheia = {
+			destroy_random_building = yes
+			destroy_random_building = yes
+			destroy_random_building = yes
+			destroy_random_building = yes
+			destroy_random_building = yes
+			location = {
+				add_province_modifier = { name = horrible_earthquake duration = -1 }
+			}
+		}
+		set_global_flag = 526_antioch_earthquake
+	}
+	option = {
+		name = hist.24.b #we must send help
+		trigger = {
+			b_antiocheia = {
+				holder_scope = { top_liege = { character = ROOT } }
+			}
+		}
+		
+		b_antiocheia = {
+			destroy_random_building = yes
+			destroy_random_building = yes
+			location = {
+				add_province_modifier = { name = horrible_earthquake duration = -1 }
```

Comment:
This province modifier seems to lack localisation and an entry in `WtWSMS_event_modifiers.txt`.

Reply:
It's localised in WtWSMS_misc.csv and exists in WtWSMS_event_modifiers.txt 🤔

Reply:
Oh, you are using the existing modifier? That explains it.

Reply:
That's my overmiss, the duration should be about a year.